scene/surface-placement/gradlew.bat

```diff
@@ -14,7 +14,7 @@ set APP_BASE_NAME=%~n0
 set APP_HOME=%DIRNAME%
 
 @rem Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-set DEFAULT_JVM_OPTS=
+set DEFAULT_JVM_OPTS="-Xmx64m"
```

Comment:
for this and the gradlew file, do we usually change these? Haven't seen these before so am just curious if this is another thing we should be remembering to set when changing/adding new samples.

Reply:
It's safe to ignore this